packages/common/client-utils/.gitignore

```diff
@@ -0,0 +1,52 @@
+# Compiled TypeScript and CSS
```

Comment:
Nit: Is all of this necessary? If not, can we trim it down to only what is needed by the package?

Reply:
Ahh, yeah this came from common-utils, so it's huge. I'll clean it up.

Reply:
I know the thread is resolved, I'm just left wondering a bit why do we even have `.gitignore` files per package when we have a repo-wide one which seems to cover all cases.

Reply:
It's part of the "lift and shift" principle that we (still sort of) follow when creating packages. Each package should be capable of being moved to somewhere else in the repo - or another repo - and it should work with minimal changes. Each package was intended to be a unit that could be moved and reorganized with little effort. The places where we no longer follow this principle, IMO, are the places that become expensive to change.